ambry-api/src/main/java/com/github/ambry/quota/QuotaMetric.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+/**
+ * Enum representing various types of quota metrics supported.
+ */
+public enum QuotaMetric {
```

Comment:
I don't think of the usage of QuotaMetric, how will it be used?

Reply:
QuotaMetric is like the unit for each quota value in `Quota` class.